h2d/Dropdown.hx

```diff
@@ -30,22 +30,54 @@ private class Fake extends Object {
 	}
 }
 
+/**
+	`h2d.Dropdown` is a simple UI component that creates an interactive drop-down list.
+	Dropdown will add an `h2d.Flow` to `Scene` when opening in order to be visible above other objects. See `dropdownLayer`.  
+	Note that when dropdownList opens and closes, item objects will recieve the `onHierarchyChanged` callback.
+**/
 class Dropdown extends Flow {
 	var items : Array<h2d.Object>;
 	var fake : Fake;
 	var cursor : h2d.Bitmap;
 	var arrow : h2d.Bitmap;
 
+	/**
+		A background Tile that is shown when user hover over an item in the dropdown list.

```

Comment:
```suggestion
		A background Tile that is shown when the user overs an item in the dropdown list.
```

Reply:
Shouldn't this be `when the user hovers over an item` instead of `when the user overs an item` ?